i18n/en-US.yml

```diff
@@ -209,6 +209,9 @@ components:
         other {# spaces available}
       }
   AppMenu:
+    agencyLogoReset: Reset trip search and return to "Plan Trip" page
```

Comment:
I do still think there needs to be a `{site title}` here, so that AT users know they're on the site logo and not just a reset trip button!

Reply:
I'm not sure how to pass a value to an `intl.formatMessage`